crown-court-contribution/src/main/java/uk/gov/justice/laa/crime/contribution/config/ResourceServerConfiguration.java

```diff
@@ -0,0 +1,76 @@
+package uk.gov.justice.laa.crime.contribution.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.annotation.Order;
+import org.springframework.http.HttpMethod;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.http.SessionCreationPolicy;
+import org.springframework.security.oauth2.jwt.JwtDecoder;
+import org.springframework.security.oauth2.server.resource.web.BearerTokenAuthenticationEntryPoint;
+import org.springframework.security.oauth2.server.resource.web.access.BearerTokenAccessDeniedHandler;
+import org.springframework.security.web.SecurityFilterChain;
+import org.springframework.security.web.access.AccessDeniedHandler;
+
+@Configuration
+@Order(1)
+@EnableWebSecurity
+public class ResourceServerConfiguration {
+
+    public static final String API_PATH = "/api/**";
+    public static final String SCOPE_READ = "SCOPE_READ";
+    public static final String SCOPE_READ_WRITE = "SCOPE_READ_WRITE";
+
+    @Bean
+    protected BearerTokenAuthenticationEntryPoint bearerTokenAuthenticationEntryPoint() {
+        BearerTokenAuthenticationEntryPoint bearerTokenAuthenticationEntryPoint = new BearerTokenAuthenticationEntryPoint();
+        bearerTokenAuthenticationEntryPoint.setRealmName("Crown Court Proceedings API");
```

Comment:
This should be Crown Court Contribution.

Reply:
Thanks. Resolved now